### PR TITLE
Package ocamlformat.0.3

### PR DESCRIPTION
**Pull request by jberdine**
### `ocamlformat.0.3`

Auto-formatter for OCaml code

OCamlFormat is a tool to automatically format OCaml code in a uniform style.

---
* Homepage: https://github.com/ocaml-ppx/ocamlformat
* Source repo: https://github.com/ocaml-ppx/ocamlformat.git
* Bug tracker: https://github.com/ocaml-ppx/ocamlformat/issues

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

File: packages/ocamlformat/ocamlformat.0.3/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/ocaml-ppx/ocamlformat/archive/0.3.tar.gz"
```
Comment (avsm) [inline packages/ocamlformat/ocamlformat.0.3/url]: Something odd happening with the release archive and `.git` being looked for...

```
#=== ERROR while installing ocamlformat.0.3 ===================================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ocamlformat -j 4
# path         /home/travis/.opam/4.06.0/build/ocamlformat.0.3
# compiler     4.06.0
# exit-code    1
# env-file     /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.env
# stdout-file  /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.out
# stderr-file  /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.err
### stderr ###
# File unavailable: .git/HEAD
```

Comment (jberdine) [reply packages/ocamlformat/ocamlformat.0.3/url]: Sorry for the noise, I missed a step in the release procedure.

**Comment by jberdine** [comment]
Note that there is really only one failure, which is only for the commit that adds a version constraint on the existing ocamlformat 0.2 package. That package is currently unbuildable on systems with broken versions of `patch`, so this is not a regression. The new package versions in this PR no longer need a working `patch`.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**